packages/design-docs/src/tokens/TokenValue.tsx

```diff
@@ -11,6 +11,7 @@ import tokens from '@talend/design-tokens';
 import dictionary from '@talend/design-tokens/lib/light/dictionary';
 import * as utils from './TokenFormatter';
 import theme from './TokenValue.module.scss';
+import { Unstyled } from '@storybook/blocks';
```

Comment:
From my understanding docs blocks can not be used in stories

Reply:
it's not a story, it's a docs block